### PR TITLE
Update external link to WordPress Coding Standards

### PR DESCRIPTION
**Pull request by feliperomero3**
Current external link goes to not found (404).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # WordPress Coding Standards Docs
 
-This repo contains the source for the [WordPress Coding Standards](https://developer.wordpress.org/coding-standards/) on https://developer.wordpress.org
+This repo contains the source for the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/installing-a-local-server/) on https://developer.wordpress.org
```
Comment (GaryJones) [inline README.md]: Surely `https://make.wordpress.org/core/handbook/best-practices/coding-standards/` would be a better URL for this?

Comment (jrfnl) [reply README.md]: See https://github.com/WordPress-Coding-Standards/docs/pull/9

Comment (feliperomero3) [reply README.md]: @GaryJones totally right! Clipboard mistake.

**Comment by ntwb** [comment]
Thanks @feliperomero3, but I'm closing this because this repo is a specific repo to sync the docs to a new site on the developer.wordpress.org domain, it is not meant to link any existing or other documentation hosted elsewhere on w.org and is waiting on further work from the #meta team to implement, once implemented the link you are seeking to change will be the correct link.

**Pull request closed without merging**